src/DotnetRelease/PatchReleaseOverviewRecords.cs

```diff
@@ -78,6 +78,10 @@ public record RuntimeComponent(
         JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
```

Comment:
I'm also not sure if the type for those with `JsonIgnore` are correct. I would assume that these should be `string?` instead.